packages/vc-verification/src/models/verifiable-credentials.ts

```diff
@@ -7,8 +7,16 @@ const { isHexString } = utils;
 
 export interface VerificationResult {
   status: boolean;
+  error: string;
 }
 
+export const verificationResult = function (
```

Comment:
I think this function isn't really needed as it just returns passed in values

Reply:
Yeah, but if in future you want to have a different format of result Object then you just need to change it at one place.

Reply:
I don't think this part is subject for changes. In any case it is not a big deal, so let's keep it now